src/finfield.c

```diff
@@ -1474,6 +1475,9 @@ Obj FuncLOG_FFE_DEFAULT (
 **  the smallest integer <i> such that '<i> * <z>^0 = <z>'.
 */
 Obj IntFF;
+#ifdef HPCGAP
+Int NumFF;
+#endif
```

Comment:
Note: the changes in here might conflict with changes in PR #2997 resp. PR by @stevelinton and @jdebeule -- we need to find a way forward there. 

I just looked a the two PRs side by side, and it *seems* to be that they do not actually overlap, so we might be fine, but it'd be good to verify this.

Reply:
Yeah, I'm aware of that PR. This is actually some code that I'd like to simplify, anyway. We have data structures that get basically initialized once and then don't get changed anymore; it should be possible to remove many the hoops I'm currently jumping through to present a consistent picture to all threads.